test/e2e/pass/multipart-formdata.har

```diff
@@ -0,0 +1,148 @@
+{
+  "log": {
+    "entries": [
+      {
+        "comment": "Request 1",
+        "checks": [],
+        "variables": [],
+        "request": {
+          "url": "http://test.k6.io/value-pairs",
+          "method": "POST",
+          "queryString": [],
+          "headers": [
+            {
+              "name": "Content-Type",
+              "value": "multipart/form-data; boundary=---boundary"
+            }
+          ],
+          "postData": {
+            "mimeType": "multipart/form-data; boundary=---boundary",
+            "comment": "keyValuePairs",
+            "params": [
+              {
+                "name": "hello",
+                "value": "world"
+              }
+            ]
+          }
+        }
+      },
+      {
+        "comment": "Request 2",
+        "checks": [],
+        "variables": [],
+        "request": {
+          "url": "http://test.k6.io/image",
+          "method": "POST",
+          "queryString": [],
+          "headers": [
+            {
+              "name": "Content-Type",
+              "value": "multipart/form-data; boundary=---boundary"
+            }
+          ],
+          "postData": {
+            "mimeType": "multipart/form-data; boundary=---boundary",
+            "comment": "",
+            "params": [
+              {
+                "fileName": "jpeg-quality-30.jpg",
+                "name": "file",
+                "value": "data:image/jpeg;base64,/9j/4AAQSkZJRgABAQEBLAEsAAD/2wBDABsSFBcUERsXFhceHBsgKEIrKCUlKFE6PTBCYFVlZF9VXVtqeJmBanGQc1tdhbWGkJ6jq62rZ4C8ybqmx5moq6T",
```

Comment:
Technically, this wouldn't be a data-uri. It's kind of redundant. But it doesn't really matter and I guess you _could_ send a data-uri in a multipart body.